guides/common/modules/proc_using-red-hat-insights.adoc

```diff
@@ -24,6 +24,16 @@ On each recommendation line, the vertical ellipsis provide a *View in {Team} Ins
 If the host has recommendations listed, click on the number of recommendations.
 On the *Insights* tab, the vertical ellipsis next to the *Remediate* button provide a *Go To {Project} Insights page* link to information for the system, and a *View in {Team} Insights* link to host details on the console.
 
+.Excluding hosts from rh-cloud and insights-client reports
+Starting from {Project} {ProductVersion}, you can omit host reporting from _rh-cloud_ and _insight-client_ reports by setting the `host_registration_insights` parameter to *False*.
```

Comment:
```suggestion
You can set the `host_registration_insights` parameter to *False* to omit host reporting from _rh-cloud_ and _insight-client_ reports.
```

We generally don't say "starting with version x" because users will always get the docs that match their Foreman/Katello/Satellite/orcharhino version.